products/zoningtaxlots/models/product/qaqc/qa_freq_changes.sql

```diff
@@ -57,7 +58,8 @@ count_new AS (
                 sum((zoning_map_code IS NOT null)::int) AS zoning_map_code
             FROM new_version
         ) AS row
-    ) AS a, json_each_text(_col) AS b
+    ) AS a
+    CROSS JOIN LATERAL json_each_text(a._col) AS b
```

Comment:
this is an interesting one!

Reply:
Ah yeah sorry, forgot about this, I should have commented. This was me. Originally, there was some weird error - sqlfluff didn't like that a didn't get mentioned at all.

The comma here is an implicit lateral cross join, something that I personally really dislike. Very unintuitive that what comes after the comma is effectively a subquery. I'd rather be explicit so that if someone doesn't understand, they could just lookup what a lateral cross join is.